### PR TITLE
zh-translation: /about/contribute/style-guide/index.md #1781

### PR DESCRIPTION
**Pull request by gorda**
- [x] Docs

Ref: [ServiceMesher#1781](https://github.com/servicemesher/istio-official-translation/issues/1781)

### CODE REVIEW

File: content/zh/about/contribute/style-guide/index.md [hunk 1]
```diff
@@ -1,144 +1,47 @@
...
+description: 介绍 Istio 文档所使用的风格约定。
+weight: 10
 aliases:
     - /zh/docs/welcome/contribute/style-guide.html
     - /zh/docs/reference/contribute/style-guide.html
 keywords: [contribute]
 ---
 
-本页提供 Istio 文档的内容准则。如果您对这些准则有更好的想法和建议，欢迎在 GitHub 中对本文档进行完善。
+Istio 文档所有的内容都必须 **清晰明了** 且 **易于理解**。我们根据 Google 开发者风格指南中的 [重点](https://developers.google.com/style/highlights) 和 [通则](https://developers.google.com/style/tone) 定义了标准。关于您贡献的的内容是如何被审核、接受的，请参见[审核页面](/zh/about/contribute/review)获取详情。
```
Comment (gauliang) [inline content/zh/about/contribute/style-guide/index.md]:
```suggestion
Istio 文档所有的内容都必须 **清晰明了** 且 **易于理解**。我们根据 Google 开发者风格指南中的 [重点](https://developers.google.com/style/highlights) 和 [通则](https://developers.google.com/style/tone) 定义了标准。关于您贡献的内容是如何被审核、接受的，请参见[审核页面](/zh/about/contribute/review)获取详情。
```

**Comment by gorda** [comment]
> 多了一个字

感谢细心 review，已修改 @gauliang

**Comment by gauliang** [comment]
@rootsongjc 
/LGTM

**Comment by gorda** [comment]
@googlebot I sign it!

**Pull request merged**